src/i_video.c

```diff
@@ -799,6 +799,9 @@ void I_FinishUpdate(void)
             {
                 I_Sleep((remaining_time - 1000) / 1000);
             }
+
+            // Play nicely with other threads while waiting
+            SDL_CPUPauseInstruction();
```

Comment:
This will need a SDL >= 2.24 build time version check.

Reply:
This is an undocumented function used in [`SDL_LockSpinlock`](https://github.com/libsdl-org/SDL/blob/57a15933cd90f46f4791faaa4394a3f8a019f234/src/atomic/SDL_spinlock.c#L171) and is not recommended by SDL developers: https://wiki.libsdl.org/SDL3/SDL_LockSpinlock#remarks Not sure about this.

Reply:
This was added to the SDL_atomic public api [here ](https://github.com/libsdl-org/SDL/commit/16e699a76155b8babb51e14d03b1c65eb6c589bb)